wsmaster/codenvy-hosted-api-system-license/src/test/java/com/codenvy/api/license/server/spi/tck/JpaSystemLicenseActionDaoTest.java

```diff
@@ -0,0 +1,169 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.api.license.server.spi.tck;
+
+import com.codenvy.api.license.server.jpa.JpaSystemLicenseActionDao;
+import com.codenvy.api.license.server.model.impl.SystemLicenseActionImpl;
+import com.codenvy.api.license.shared.model.Constants;
+import com.google.common.collect.ImmutableMap;
+import com.google.inject.Inject;
+import org.eclipse.che.api.core.ConflictException;
+import org.eclipse.che.api.core.NotFoundException;
+import org.eclipse.che.api.core.notification.EventService;
+import org.eclipse.che.commons.test.tck.TckListener;
+import org.eclipse.che.commons.test.tck.repository.TckRepository;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+import static com.codenvy.api.license.shared.model.Constants.Action.ACCEPTED;
+import static com.codenvy.api.license.shared.model.Constants.Action.ADDED;
+import static com.codenvy.api.license.shared.model.Constants.Action.EXPIRED;
+import static com.codenvy.api.license.shared.model.Constants.PaidLicense.FAIR_SOURCE_LICENSE;
+import static com.codenvy.api.license.shared.model.Constants.PaidLicense.PRODUCT_LICENSE;
+import static java.util.Arrays.asList;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+/**
+ * @author Anatolii Bazko
+ */
+@Listeners(TckListener.class)
+@Test(suiteName = "systemLicenseActionDaoTck")
+public class JpaSystemLicenseActionDaoTest {
```

Comment:
Please, rename it to `SystemLicenseActionDaoTest`. Because it is Tck that test dao interface and jpa is one of the of possible implementations